tools/sigh.js

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
```

Comment:
Just to make sure I'm not missing a reference, by `sigh` then you mean only this self-contained thing, which does not involve this [sigh](https://github.com/sighjs/sigh) at all?

Reply:
No, not at all related to that -- should have known there would already be something of the same name.

I just had threatened that if I ever had to write a JavaScript build framework that I would call it `sigh`, both because it's an onomatopoeia and because it accurately describes the state of the JavaScript build framework ecosystem.